ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.cpp

```diff
@@ -461,7 +461,7 @@ void ValidateUtilityNetworkTopology::onValidate()
         m_busy = false;
         emit isBusy();
       }
-      else
+      else if(status == JobStatus::Failed)
```

Comment:
With `if` here, that means other enum values are not handled. I guess it's fine for `NotStarted`, `Started`, and `Paused`, but maybe the value `Canceling` should be handled.

Reply:
Thanks @GuillaumeBelz , I have made the changes as per your suggestion